src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -291,9 +290,8 @@ module.exports = function<T, P, I, TI, PI, C, CX, PL>(
       priorityContext = nextPriorityLevel;
 
       if (
-        didCommit &&
-        (nextPriorityLevel === TaskPriority ||
-          nextPriorityLevel === SynchronousPriority)
+        nextPriorityLevel === TaskPriority ||
+        nextPriorityLevel === SynchronousPriority
```

Comment:
...well that example is bad now that the check is in `findNextUnitOfWork`. Here's a better one: if you commit at async priority, then you have remaining time, so you commit again at a lower priority, that's not a nested update, it's just two separate priority levels. By adding this check, we know we're only counting updates that were scheduled in the previous commit phase.

The other way we could implement this is by counting the number of commits in which `setState` is called at least once. Then we don't have to do a priority check. Now that I think about it, that might be more elegant.